NitroxModel/Packets/CyclopsChangeEngineMode.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace NitroxModel.Packets
+{
+    [Serializable]
+    public class CyclopsChangeEngineMode : AuthenticatedPacket
+    {
+        public String Guid { get; private set; }
```

Comment:
We removed all the private set functions; they are not needed if the value is only initialized in the constructor, and we don't (want) to change them anyway.